content/hi/portability.md

```diff
@@ -0,0 +1,9 @@
+---
+title: पोर्टेबिलिटी (Portability)
+status: Completed
+category: गुण
+---
+
+एक सॉफ्टवेयर विशेषता, पोर्टेबिलिटी पुन: प्रयोज्यता (reusability) का एक रूप है जो कुछ ऑपरेटिंग वातावरण में "अवरुद्ध" (लॉक-इन) से बचने में मदद करती है, जैसे क्लाउड प्रदाता, ऑपरेटिंग सिस्टम(os) या विक्रेता (vendors)।
```

Comment:
we should capitalize (os) to (OS)

Reply:
replace with `वातावरणों` to match plural `systems`

Reply:
Updated :)